pkg/rootless/rootless_linux.go

```diff
@@ -68,35 +68,35 @@ func tryMappingTool(tool string, pid int, hostID int, mappings []idtools.IDMap)
 }
 
 // BecomeRootInUserNS re-exec podman in a new userNS
```

Comment:
Can you expand the comment to explain what's being returned? Not entirely obvious at first glance that the int is a return code